cli/Makefile

```diff
@@ -35,6 +35,9 @@ endif
 	goreleaser build --single-target --rm-dist --snapshot -f ../.goreleaser.yaml
 	cp `cat ./dist/artifacts.json | jq -rc '.[0].path'` ./dist/tracetest
 
+simple-build:
+	go build -o tracetest main.go
```

Comment:
you can remove this. i've updated the `build` target so it only builds the local target, but still uses the goreleaser config